Makefile

```diff
@@ -73,22 +73,26 @@ TMP_FILES := bpf/kmesh/bpf2go/bpf2go.go \
 	bpf/kmesh/ads/include/config.h \
 	bpf/include/bpf_helper_defs_ext.h
 
-.PHONY: all install uninstall clean build docker
+.PHONY: all install uninstall clean build docker dataplane controller
 
-all:
-	$(QUIET) find $(ROOT_DIR)/mk -name "*.pc" | xargs sed -i "s#^prefix=.*#prefix=${ROOT_DIR}#g"
+all: dataplane controller
```

Comment:
What are the benefits of doing so? Compile the control plane go program separately without having some dependencies on the kernel, I guess?

Also we should also update dev doc, tell users that they can do this.

Reply:
This is supposed to make it easier to test without having to compile a daemon.

Reply:
This is a user requirement. They need the kernel capabilities related to the ebpf of the kmesh. Then, they make some customized modifications on the control plane. Therefore, the compilation is split.